site/daemon.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Weave Flux Daemon
+menu_order: 90
+---
+
+# Weave Cloud Deploy
```

Comment:
Though it is true that flux is the basis of 'Deploy, we ought to at least lead with OSS-oriented reference information here. In other words, this can just talk about the flux daemon on its own terms.

Reply:
The intention was to put fluxd into the Weave Cloud context, but you are absolutely right, that it is an independent beast. Done.